lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinition.cs

```diff
@@ -282,5 +285,10 @@ internal IConfiguration ResultHandlerConfiguration
         {
             get { return _resultHandlerConfiguration; }
         }
+
+        /// <summary>
+        /// Configuration for job cancellation signal implementation
+        /// </summary>
+        internal IConfiguration JobCancelSignalConfiguration { get; private set; }
```

Comment:
Use _jobCancelSignalConfiguration as an instance variable instead of a property to be consistent with others.